src/impl/peerconnection.hpp

```diff
@@ -157,6 +159,8 @@ struct PeerConnection : std::enable_shared_from_this<PeerConnection> {
 
 	Queue<shared_ptr<DataChannel>> mPendingDataChannels;
 	Queue<shared_ptr<Track>> mPendingTracks;
+
+	std::vector<struct RemoteFingerprint> rFingerprints;
```

Comment:
Private member variables should be prefixed by `m`. Actually, since a single fingerprint is passed to the verifier callback, you can store only the one fingerprint:
```suggestion
    CertificateFingerprint::Algorithm mRemoteFingerprintAlgorithm = CertificateFingerprint::Algorithm::Sha256;
	optional<string> mRemoteFingerprint;
```

Reply:
Done!